cmd/ci-operator-prowgen/main.go

```diff
@@ -399,18 +401,25 @@ func generatePresubmitForTest(name string, info *config.Info, podSpec *kubeapi.P
 	}
 
 	newTrue := true
+	return prowconfig.JobBase{
+		Agent:  "kubernetes",
+		Labels: labels,
+		Name:   jobName,
+		Spec:   podSpec,
+		UtilityConfig: prowconfig.UtilityConfig{
+			DecorationConfig: &v1.DecorationConfig{SkipCloning: &newTrue},
+			Decorate:         true,
+		},
+	}
+}
 
+func generatePresubmitForTest(name string, info *config.Info, podSpec *kubeapi.PodSpec) *prowconfig.Presubmit {
+	if len(info.Variant) > 0 {
+		name = fmt.Sprintf("%s-%s", info.Variant, name)
```

Comment:
Why is this more appropriate outside of `generateJobBase`?

Reply:
We need to have this string on this level to make the Reporter/RerunCommand/Trigger properties of presubmits (L425-L429).